scripts/src/main/resources/scripts/command/eclipse

```diff
@@ -430,18 +464,20 @@ do
   elif [ "${1}" = "add-plugin" ]
   then
     shift
-    doSetup silent
     doAddPlugin "${@}"
     exit ${?}
-  elif [ "${1}" = "import" ] 
+  elif [ "${1}" = "remove-plugin" ]
+  then
+    shift
+    doRemovePlugin "${@}"
+    exit ${?}
+  elif [ "${1}" = "import" ]
   then
-    doSetup silent
```

Comment:
if you remove this here, you should call it from the function calling the `${ECLIPSE} ...` command.

Reply:
readded doSetup silent